css/media.css

```diff
@@ -100,6 +124,11 @@
     
     }
 
+/* width doesn't change for desktop and tablet 
+    #skills ul {
+        width: 40%;  
+    }
+*/
```

Comment:
Do you mean this doesn't work? Personally, I'd remove any code that's not being used before submitting a PR.